site/docs/embed.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+menu: docs
+title: Embed
+permalink: /docs/embed.html
+---
+
+This Document assumes that you have a Vega-Lite spec as a JavaScript object in a variable called `vlSpec`. You can find a simple example in our [tutorial]({{site.baseurl}}/tutorials/getting_started.html#embed).
```

Comment:
For simple things like these, can you just fix it? 

Also, yes, specification everywhere rather than spec.